docs/source/quickstart-gcp.rst

```diff
@@ -49,20 +49,20 @@ As you work through this quickstart, you may occasionally see a message from the
 Get ElasticBLAST
 ----------------
 
-Copy and paste the commands below at the Cloud Shell prompt to install ElasticBLAST.
+Copy and paste the commands below at the CloudShell prompt to install
+ElasticBLAST.
 
-.. code-block:: shell
+.. code-block:: bash
 
-    sudo apt-get install python-wheel
-    sudo pip3 install --upgrade setuptools
-    sudo pip3 install --upgrade pip
-    sudo pip3 install elastic-blast
+    [ -d .elb-venv ] && rm -fr .elb-venv
+    python3 -m venv .elb-venv
+    source .elb-venv/bin/activate
+    pip install elastic-blast=={VERSION}
```

Comment:
Also in the interest if keeping things simple for a user, why not add `pip install wheel` here. Currently one must do it in GCP cloud shell, so every user will get an error right in the beginning. I am not sure if it is also needed in AWS cloud shell.

Reply:
Thanks, this is done in 2206fdb . I tested this in my GCP account's cloudshell and it worked well. I also tested this on my AWS account's cloudshell and `pip install wheel` was not needed.